protos/vehicle_data.proto

```diff
@@ -216,6 +227,188 @@ message LocationValue {
   double longitude = 2;
 }
 
+enum FollowDistanceEnum {
```

Comment:
`FollowDistanceEnum` -> `FollowDistance` ?

Reply:
`Enum` was added to the end of enums due to namespace conflicts.